spyder/utils/conda.py

```diff
@@ -145,6 +160,14 @@ def get_list_conda_envs():
         path = osp.join(env, 'python.exe') if WINDOWS else osp.join(
             env, 'bin', 'python')
 
+        # Skip if executable does not exist
+        if not osp.exists(path):
+            continue
+
+        # Skip if conda-based Spyder runtime or base environment
```

Comment:
Why don't we list the base env among the available envs anymore?

Reply:
Only the base environment of the conda-based installer is omitted. Base environments in general are not affected. Spyder's runtime environment for conda-based installers is still used for "Same as Spyder", but not listed in the "custom" Python interpreters. I don't think we should be advertising these environments to users, inviting them to manipulate them and break Spyder.